libs/core/src/components/pds-checkbox/pds-checkbox.scss

```diff
@@ -1,21 +1,20 @@
 :host {
-  --background-color: var(--pine-color-base-white);
-  --background-color-disabled: var(--pine-color-neutral-grey-200);
-  --background-color-hover: var(--pine-color-neutral-grey-100);
-  --border: var(--pine-border-interactive);
-  --border-color-disabled: var(--pine-color-neutral-charcoal-100);
-  --border-color-hover: var(--pine-border-interactive-hover-color);
-  --border-color-icon: var(--pine-color-base-white);
-  --border-radius: var(--pine-border-radius-xs);
+  --border-default: var(--pine-border-width-thin) solid var(--pine-color-grey-400);
+  --border-color-interactive-hover: var(--pine-border-width-thin) solid var(--pine-color-grey-500);
+  --border-radius: var(--pine-border-radius-050);
   --box-shadow-focus: 0 0 0 2px var(--pine-color-primary-200);
   --box-shadow-focus-invalid: 0 0 0 2px var(--pine-color-red-200);
-  --color-checked: var(--pine-color-neutral-charcoal-500);
-  --color-disabled: var(--pine-color-neutral-grey-300);
+  --color-background: var(--pine-color-white);
+  --color-background-disabled: var(--pine-color-grey-200);
+  --color-background-hover: var(--pine-color-grey-100);
+  --color-border-disabled: var(--pine-color-charcoal-100);
+  --color-border-icon: var(--pine-color-white);
+  --color-checked: var(--pine-color-charcoal-500);
+  --color-disabled: var(--pine-color-grey-300);
   --color-focus: var(--pine-color-primary-200);
```

Comment:
Could we add more context here for these?

Reply:
@anechol so for the --color-checked and --color-disabled those apply both a background and a border color. let me know how i should handle those.